### PR TITLE
Disable cache

### PR DESCRIPTION
**Pull request by CalebMorris**

### CODE REVIEW

**Comment by CalebMorris** [comment]
@moroshko Is there anything else I should add to this PR?

File: README.md [hunk 1]
```diff
@@ -291,6 +292,13 @@ When rendering multiple `<Autosuggest />`s, make sure to give them unique `id`s.
 <Autosuggest id="destination" suggestions={getDestinationSuggestions} />
 ```
 
+<a name="disableCacheOption"></a>
+#### scrollBar (optional)
```
Comment (moroshko) [inline README.md]: copypasta is back?

File: test/Autosuggest.js [hunk 2]
```diff
@@ -1093,4 +1093,28 @@ describe('Autosuggest', () => {
...
   });
+
+  describe('(disabled caching)', () => {
+    beforeEach(() => {
+      createAutosuggest(<Autosuggest disableCache={true} suggestions={getSuburbs} />);
+      setInputValue('m');
+      getSuburbs.reset();
+    });
+
+    describe('should not call suggestions function if', () => {
```
Comment (moroshko) [inline test/Autosuggest.js]: not call => call ?

**Comment by moroshko** [comment]
@CalebMorris Except these minor things, all looks good!

**Comment by moroshko** [comment]
Available on npm in 1.18.0 now

**Pull request merged**